packages/aardvark-react/src/default_landing_page.tsx

```diff
@@ -126,6 +169,20 @@ export class DefaultLanding extends React.Component<DefaultLandingProps, Default
 
 	}
 
+	@bind
+	private async getSettingsForGadget()
+	{
+		try
+		{
+			let promResult: AvGadgetSettings = await this.gadgetList.current.getSettingsForGadget( this.gadgetUrl );
+			this.setState( { hostSettings: promResult } );
```

Comment:
If you rename `promResult` -> `hostSettings` you can omit everything after the colon on the setState.

You shouldn't need the type on promResult either. Not as long as `getSettingsForGadget()` returns `Promise< AvGadgetSettings>` anyway.